WRIO-InternetOS/start.js

```diff
@@ -0,0 +1,187 @@
+var cssUrl = require('./js/global').cssUrl,
```

Comment:
@Xurma2 I would suggest to rename this file to preloader.js or something like that, to not confuse with start.js